libs/platform/osutils/windows/rmdir.c

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2024, Philip Meulengracht
+ *
+ * This program is free software : you can redistribute it and / or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation ? , either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ * 
+ */
+
+#include <chef/platform.h>
+#include <stdio.h>
+#include <Windows.h>
+
+
+int platform_rmdir(const char *path) {
+    WIN32_FIND_DATA find_file_data;
+    HANDLE hFind;
+    char full_path[MAX_PATH];
+    int r = 0;
```

Comment:
Du mikser alt for meget coding styles - det er rigtig fint at kopiere kode fra andre steder men please gør den konsistent
```suggestion
    WIN32_FIND_DATA findFileData;
    HANDLE hFind;
    char fullPath[MAX_PATH];
    int result = 0;
```